packages/shared/src/lib/featureValues.ts

```diff
@@ -23,10 +23,16 @@ export interface FeatureTheme {
   light?: FeatureThemeVariant;
   dark?: FeatureThemeVariant;
   cursor?: string;
+
   [key: string]: number | FeatureThemeVariant | string | undefined;
 }
 
 export enum SeoSidebarExperiment {
   Control = 'control',
   V1 = 'v1',
 }
+
+export enum UpvoteExperiment {
+  Control = 'control',
+  Animated = 'v1',
+}
```

Comment:
I don't think this needs to be a enum feature, can simply be boolean if it's on/off :)

Reply:
Yeah I hadn't specified in the ticket and hadn't created the experiment yet, to not make Luca change anything since he was already done with this just went and created it like this.

Reply:
yeah sure